bool is_localized
float64 localization_accuracy
Location location
